feeds/product.php

```diff
@@ -270,24 +270,18 @@ function arrayMergeByIdProduct($array1 = [], $array2 = [])
 
 $products = DfTools::getAvailableProducts($lang->id, $shouldShowProductVariations, $limit, $offset);
 foreach ($products as $product) {
-    $minProductPrices = [];
+    $minPriceVariant = null;
     if ($shouldShowProductVariations && $product['variant_count'] > 0) {
         $variations = DfTools::getProductVariations($product['id_product']);
         foreach ($variations as $variation) {
-            if ($shouldDisplayPrices) {
-                $variantPrices = DfTools::getVariantPrices($variation['id_product'], $variation['id_product_attribute'], $shouldPricesUseTaxes, $currencies);
-                if (!isset($minProductPrices['onsale_price']) || $variantPrices['onsale_price'] < $minProductPrices['onsale_price']) {
-                    $minProductPrices = $variantPrices;
-                }
-            }
-            $expanded_variation = array_merge($product, $variation);
-            $built_variation = $dfProductBuild->buildProduct($expanded_variation, [], $additionalAttributesHeaders, $additionalHeaders);
+            $minPriceVariant = $dfProductBuild->getMinPrice($minPriceVariant, $variation);
+            $built_variation = $dfProductBuild->buildVariation($product, $variation);
             $csv_product = $dfProductBuild->applySpecificTransformationsForCsv($built_variation, $extraHeader, $header);
             fputcsv($csv, $csv_product, DfTools::TXT_SEPARATOR);
```

Comment:
Well, this one is not part of the PR, just a proposal:
```suggestion
            fputcsv($csv, $csvProduct, DfTools::TXT_SEPARATOR);
```